front/pages/w/[wId]/builder/data-sources/new.tsx

```diff
@@ -59,8 +60,9 @@ export default function DataSourceNew({
   dataSources,
   gaTrackingId,
 }: InferGetServerSidePropsType<typeof getServerSideProps>) {
-  const [disabled, setDisabled] = useState(true);
-  const [creating, setCreating] = useState(false);
+  const [isSaving, setIsSaving] = useState(false);
+  const [edited, setEdited] = useState(false);
+  const [valid, setValid] = useState(true);
```

Comment:
Nit consistency: adding `is` prefix for all or for none? (I vote for having the prefix so we instantly know it's a boolean)

Reply:
Done :+1: